docs/topics/running/host-crd.md

```diff
@@ -126,6 +126,8 @@ requestPolicy:
     additionalPort: insecure-port
 ```
 
+> **WARNING:** `requestPolicy` is applied globally, even if it is applied to only one `Host` and you have multiple `Host`s configured. Different `requestPolicy` behaviors cannot be applied to different `Host`s.   A `requestPolicy` defined on one `Host` will override the default behavior on another `Host` with no `requestPolicy` defined.
```

Comment:
I think there should be guidance in telling people that if they have `requestPolicy` that they want set, to copy it on to ALL Host resources for that Ambassador installation rather than having just one dictate and overwrite the policy of others (since this behavior is not always obvious)